lib/ui/src/components/sidebar/Tree.tsx

```diff
@@ -243,6 +246,19 @@ const Node = React.memo<NodeProps>(
           setExpanded({ ids: [item.id], value: !isExpanded });
           if (item.type === 'component' && !isExpanded) onSelectStoryId(item.id);
         }}
+        onMouseEnter={() => {
+          if (item.isComponent) {
+            api.emit(PRELOAD_STORIES, {
+              ids: [item.children[0]],
+              options: {
+                target:
+                  refId && refId !== 'storybook_internal'
+                    ? `storybook-ref-${refId}`
+                    : 'storybook-preview-iframe',
+              },
+            });
+          }
```

Comment:
Can we add `api.preloadEntries()` to the stories module? Also we could consider renaming the event to `PRELOAD_ENTRIES` as they don't necessarily need to be stories (they can be docs)

Reply:
I renamed the event-name..

Are you suggesting changing this:
```ts
            api.emit(PRELOAD_ENTRIES, {
              ids: [item.isLeaf ? item.id : item.children[0]],
              options: { target: refId },
            });
```
to:
```ts
            api.preloadEntries({
              ids: [item.isLeaf ? item.id : item.children[0]],
              options: { target: refId },
            });
```
?

Reply:
If that's all it's doing then I guess I agree with your (implied) point that it's not worth it!